### PR TITLE
Add zone-outage scenario to run in prow CI

### PR DESCRIPTION
**Pull request by chaitanyaenr**

### CODE REVIEW

File: prow/zone-outages/prow_run.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# Move kraken from root dir to tmp to avoid permissions issues in prow until fixed in base image
+cp -r /root/kraken /tmp/kraken
+krkn_loc=/tmp/kraken
+
+# Substitute config with environment vars defined
+envsubst < zone-outages/zone_outage_scenario.yaml.template > $krkn_loc/scenarios/zone_outage.yaml
+envsubst < $krkn_loc/config/config.yaml.template > $krkn_loc/config/zone_config.yaml
+
+export SCENARIO_TYPE=zone_outages
```
Comment (paigerube14) [inline prow/zone-outages/prow_run.sh:31]: do we need to set the scenario type? don't we want the default from the env file? https://github.com/redhat-chaos/krkn-hub/blob/main/zone-outages/env.sh

File: prow/zone-outages/prow_run.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+oc config view
+echo "Printing node info"
+for node in $(oc get nodes | awk 'NR!=1{print $1}'); do oc get node/$node -o yaml; done
+
+# Move kraken from root dir to tmp to avoid permissions issues in prow until fixed in base image
+cp -r /root/kraken /tmp/kraken
+krkn_loc=/tmp/kraken
+
+# Substitute config with environment vars defined
+envsubst < zone-outages/zone_outage_scenario.yaml.template > $krkn_loc/scenarios/zone_outage.yaml
```
Comment (paigerube14) [inline prow/zone-outages/prow_run.sh:28]: I think we need to make the file local to krkn-hub, krkn_loc I believe isn't writable

https://github.com/redhat-chaos/krkn-hub/blob/61b698ebb03c8e919bec5a813eafe281d860511c/prow/namespace-scenarios/prow_run.sh#L26

File: prow/zone-outages/prow_run.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+envsubst < $krkn_loc/config/config.yaml.template > $krkn_loc/config/zone_config.yaml
```
Comment (paigerube14) [inline prow/zone-outages/prow_run.sh:29]: I think we need to edit the krkn-hub config file similar to [here](https://github.com/redhat-chaos/krkn-hub/blob/61b698ebb03c8e919bec5a813eafe281d860511c/prow/namespace-scenarios/prow_run.sh#L27) and make it local like above comment

File: prow/zone-outages/prow_run.sh (hunk 1 above)
Comment (chaitanyaenr) [reply prow/zone-outages/prow_run.sh]: Fixed it. Thanks.

File: prow/zone-outages/prow_run.sh (hunk 2 above)
Comment (chaitanyaenr) [reply prow/zone-outages/prow_run.sh]: Fixed it. Thanks.

File: prow/zone-outages/prow_run.sh (hunk 3 above)
Comment (chaitanyaenr) [reply prow/zone-outages/prow_run.sh]: Fixed it. Thanks.

**Review by paigerube14 — APPROVED** [review]
/lgtm

**Pull request merged**